### PR TITLE
docs: replace `LocalUtxo` to `LocalOutput`

### LINKED ISSUE #1249: Better names for methods
It looks like `LocalUtxo` was renamed to `LocalOutput` in master, which seems like a great improvement in clarity. I came across #665 trying to determine whether or not `Wallet::get_utxo` actually returns *unspent* outputs or not. I'd propose renaming the rest of the wallet API (and database API) to reflect this type change in 1.x

### PR DESCRIPTION
**Pull request by 15IITian**
#### All Submissions:

* [x] I followed the [contribution guidelines](https://github.com/bitcoindevkit/bdk/blob/master/CONTRIBUTING.md)
* [x] I ran `cargo fmt` and `cargo clippy` before committing

 fixes #1249

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -205,7 +205,7 @@ BDK and LDK together.
 - `verify` flag removed from `TransactionDetails`.
 - Add `get_internal_address` to allow you to get internal addresses just as you get external addresses.
 - added `ensure_addresses_cached` to `Wallet` to let offline wallets load and cache addresses in their database
-- Add `is_spent` field to `LocalUtxo`; when we notice that a utxo has been spent we set `is_spent` field to true instead of deleting it from the db.
```
Comment (danielabrozzoni) [inline CHANGELOG.md]: I don't think we should be modifying the old changelog - at the time, when we published those versions, LocalOutput was indeed called LocalUtxo :) 

Can you please revert your changes on the CHANGELOG.md file?

Comment (15IITian) [reply CHANGELOG.md]: I got  your point .

**Comment by notmandatory** [comment]
@Ademan is this what you had in mind for #1249 ? If so It's a very small and non-functional change and I'd prefer it be done as basic house keeping in some other more substantial PR.

**Comment by 15IITian** [comment]
>  It's a very small and non-functional change and I'd prefer it be done as basic house keeping in some other more substantial PR.

If so we should close it .

**Comment by notmandatory** [comment]
Yes I'll close this PR and someone can fix it as housekeeping in another PR.

**Pull request closed without merging**